.github/actions/gali/README.md

```diff
@@ -0,0 +1,11 @@
+# `gali`
```

Comment:
Cool stuff, we want to back our processes with automation.

Is this meant to be incubation for a `gali` action that will be hosted in its own repository some day?